e2etests/web/regular_integration_tests/lib/treeshaking_main.dart

```diff
@@ -1,25 +1,26 @@
-
```

Comment:
How did this survive the license header check?

Reply:
But also, we're less strict about this code. It is exempt from the license scraper (see https://github.com/flutter/engine/pull/26659).